src/utils/keywordGenerator.js

```diff
@@ -0,0 +1,54 @@
+import { cleanDescription, cleanStopWords } from './CleanDescription.js';
```

Comment:
```suggestion
import { cleanDescription, cleanStopWords } from './CleanDescription';
```
you don't have to add .js part in js imports, can you check all of your js imports and if there are .js in the file paths can you delete it